app/models/event.rb

```diff
@@ -32,7 +32,7 @@ class Event < ApplicationRecord
   validates :name, :discipline, :game_mode, presence: true
   validates :name, :discipline, :game_mode, :player_type, presence: true
   validates :deadline, :startdate, :enddate, presence: true
-  validates :max_teams, numericality: { greater_than_or_equal_to: 0 } # this validation will be moved to League.rb once leagues are being created and not general event objects
+  validates :max_teams, numericality: { greater_than_or_equal_to: 0, allow_nil: true }
```

Comment:
Will sooner or later be moved to the league and tournament models, since a rankinglist doesn't havea maximum amount of teams I think.